src/main/java/core/basesyntax/model/Figure.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.model;
+
+import core.basesyntax.interfaces.AreaCalculator;
+import core.basesyntax.interfaces.Drawable;
+
+public abstract class Figure implements AreaCalculator, Drawable {
+
+    protected String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
```

Comment:
Please don't add this redundant empty line.

Reply:
Fixed